etc/smashbox.conf.template

```diff
@@ -131,3 +131,27 @@ reset_server_log = False
 #
 # scp port to be used in scp commands, used primarily when copying over the server log file
 scp_port=22
+
+from collections import OrderedDict
```

Comment:
Each item of the dictionary is a definition of the processor object (from  python/smashbox/configgen/processors.py) that needs to be created. The key is the class, and the value is the parameters to create the object.

The default dictionary implementation doesn't guarantee the traversal order, and that's something we need to insert the objects in the proper order (as defined below). The orderedDict implementation guarantees the order.

The other option is the create a list of objects, but it's seems a bit more complex than what we need. I don't think we'll need changes here in the short term, unless we need to apply several processors of the same type (which is not possible right now)